osmosis-1/osmosis.zone_assets.json

```diff
@@ -2719,6 +2719,14 @@
       "path": "transfer/channel-20100/nscr",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "base_denom": "ubedrock",
+      "chain_name": "pylons",
+      "path": "transfer/channel-17863/ubedrock",
```

Comment:
There is a typo in the channel. Is it 17863 or 17683?

Reply:
Looks like 17683 in my Keplr